source/administration/config-settings.rst

```diff
@@ -3027,6 +3027,15 @@ A 32-character key for encrypting and decrypting sensitive fields in the databas
 
 When using High Availability, the salt must be identical in each instance of Mattermost.
 
+The following fields are encrypted using this key
+- ``SqlSettings.DriverName``
+- ``SqlSettings.DataSource``
+- ``SqlSettings.MaxIdleConns``
+- ``SqlSettings.MaxOpenConns``
+- ``SqlSettings.Trace``
+- ``SqlSettings.QueryTimeout``
+- ``SqlSettings.ConnMaxLifetimeMilliseconds``
```

Comment:
This list doesn't show up as a list on preview (the items on the list appear in one line). Maybe this will be fixed by adding a space between the header and the list (similar to what you did below).